internal/bminventory/inventory.go

```diff
@@ -4237,6 +4237,9 @@ func (b *bareMetalInventory) uploadLogs(ctx context.Context, params installer.Up
 		if err != nil {
 			return err
 		}
+		b.eventsHandler.AddEvent(ctx, params.ClusterID, params.HostID, models.EventSeverityInfo,
+			fmt.Sprintf("uploaded logs for host %s cluster %s to bucket %s",
+				params.HostID.String(), params.ClusterID.String(), b.S3Bucket), time.Now())
```

Comment:
Is `b.S3Bucket` useful for the user? What will be the value when not using S3 but filesystem?

Reply:
+1 i think that a simple message like `logs are uploaded for host <hostname>` is enough information for the user.

Reply:
done